packages/corvid-local-server/test/it/localDocumentChanges.spec.js

```diff
@@ -91,7 +97,7 @@ describe("local changes", () => {
           onDocumentChange
         );
 
-        let filePath = localSiteBuilder.getLocalFilePath(docItem);
+        const filePath = localSiteBuilder.getLocalFilePath(docItem, "page");
```

Comment:
asking for "page" is only correct if the itemKey is page/pageWithCode/lightbox/lightboxWithCode